apps/src/music/player/ProgramSequencer.js

```diff
@@ -32,6 +32,13 @@ export default class ProgramSequencer {
     this.stack.push({measure: measure, together: false});
   }
 
+  // Beginning of a play_sequential block, but with an explicit starting measure.
+  // Should only be used for an outermost sequential block, typically in an trigger
+  // handler.
+  playSequentialWithMeasure(currentMeasure) {
```

Comment:
I wonder if it would be clearer for this to be something like `setCurrentMeasure(currentMeasure, isSequential)`? That way it's more indicative of the fact that you are manually the current position of the sequencer?

Reply:
Yeah, I was kind of on the fence about this.  One thing I was trying to achieve by making it a `playSequential...` function was to continue the pattern of enclosing these blocks with a `play...` and then a matching `end...` call.

Reply:
True, yeah that makes sense. I was actually wondering if we could break this into two calls, `setMeasure(measure)` and then `playSequential()` so you still get the matching pairs of `play...` and `end...` calls, but seems like that may be overkill as it may require storing extra information in the class outside of the stack.

Reply:
Yeah, I think it makes sense to keep as a single function in its current form because the measure is stored in the stack itself.  Happy to revisit if/when we continue evolving this model.